extensions/mssql/package.nls.json

```diff
@@ -31,6 +31,7 @@
 	"title.clearSearchServerResult": "Search: Clear Search Server Results",
 
 	"title.endpoints": "Service Endpoints",
+	"title.books": "Books",
```

Comment:
Let's quickly run this E2E with PM, we should get their help to tweak these names

Reply:
Per @vickyharp , let's call this "Notebooks" for right now